doc/queries.md

```diff
@@ -395,11 +397,148 @@ Result:
 <a name="aggs-nested"></a>
 
 ### 4. Nested Aggregation
-Guppy supports nested aggregations (sub-aggregations) for fields. Currently Guppy only supports two-level-sub-aggregations.
+:bangbang: **This section is for performing aggregations on documents which contain nested fields. For information about Guppy supporting nested sub-aggregations such as terms aggregation and missing aggregation, please refer to [Sub-aggregations](#aggs-sub)**
 
-There are two types of nested aggregations that is supported by Guppy: terms aggregation and missing aggregation, user can mix-and-match the using of both aggregations.
+Guppy supports performing aggregations (both text and numeric aggregations) on nested fields. For information about using nested fields inside filters, see [Nested Filter](#filter-nested)
```

Comment:
Could you add a quick paragraph explaining the difference between sub-aggregations and nested aggregations? I'm having a hard time understanding what the difference is reading the docs.

Reply:
Also, is the nested aggregation also limited to a depth of 2 levels?

Reply:
> Also, is the nested aggregation also limited to a depth of 2 levels?

no not really, it should work for more nested levels